package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.5.20",
+  "version": "3.5.21",
```

Comment:
I'm not sure why GH isn't showing a merge conflict here, since another PR has already updated this version. (The left side of the diff shows main branch, which isn't 20 any more.)

I recommend updating the version number to 22 so we can track your change through Dev/Test/Prod environments.